pilot/pkg/kube/inject/inject.go

```diff
@@ -175,6 +175,14 @@ func (p *Params) Validate() error {
 type Config struct {
 	Policy InjectionPolicy `json:"policy"`
 
+	// AlwaysEnabledLabels: Pods that contain any of those labels always
+	// get the sidecar injected
+	AlwaysEnabledLabels []string `json:"always-enabled-labels"`
+
+	// AlwaysDisabledLabels: Pods that contain any of those labels never
+	// get the sidecar injected
```

Comment:
For completeness, mention that disabled overrides enabled if present in both lists.